src/test/java/org/jenkinsci/plugins/workflow/support/pickles/ExecutorPickleTest.java

```diff
@@ -126,52 +114,6 @@
         });
     }
 
-    /** Ephemeral and deleted when it disconnects */
-    static class EphemeralDumbAgent extends Slave implements EphemeralNode {
```

Comment:
I was confused about this earlier as well, looks like it was added in #47, but then the test using it was deleted in #48, but the class stayed around unused.

Reply:
Indeed. Fun fact: #48 was reverting #47 because it broke a production use case for me!